nodelet/include/nodelet/nodelet.h

```diff
@@ -127,6 +183,13 @@ class Nodelet
   // Protected data fields for use by the subclass.
 protected:
   inline const std::string& getName() const { return nodelet_name_; }
+  inline const std::string getSuffixedName(const std::string& suffix) const
```

Comment:
Since the function can't return a reference the const keyword for the return type seems to be unnecessary.